extras/secure-email.rst

```diff
@@ -1,59 +1,78 @@
 ﻿Secure Email
 ============
 
-Zammad supports S/MIME for high-security email communication. 
+Zammad supports two systems of high-security email communication:
+   * Pretty Good Privacy (PGP) 
+   * Secure/Multipurpose Internet Mail Extensions (S/MIME).
 
 .. figure:: /images/extras/secure-email/creating-articles_signed-and-encrypted.gif
    :alt: Screencast demo of S/MIME features for both new tickets and replies
    :scale: 50%
    :align: center
 
-   Use the 🔒 **Encrypt** and ✅ **Sign** buttons to turn on encryption and signing for outgoing emails.
+   Use the 🔒 **Encrypt** and ✅ **Sign** buttons to turn on encryption and
+   signing for outgoing emails.
 
-.. note:: **🤔 Huh? I don’t see “Sign” or “Encrypt” options in the ticket view...** 
+.. note:: **🤔 Huh? I don’t see “Sign” or “Encrypt” options in the ticket 
+   view...** 
 
    This feature is **optional**;
    if you don’t see it in the ticket composer,
    that means your administrator hasn’t enabled it yet.
-   Administrators can learn more
-   `here <https://admin-docs.zammad.org/en/latest/system/integrations/smime.html>`_.
+   Administrators can learn more here:
+      * :admin-docs:`PGP </system/integrations/pgp/index.html>`
+      * :admin-docs:`S/MIME </system/integrations/smime/index.html>`
```

Comment:
These two lines have been indented in a wrong way, please check. (Hint: `make html` will spit out an error regarding this problem.)

Reply:
The "correct" indenting broke the list. I inserted an empty line above and it looks like it works now :-)